src/main/python/interfaces/settings/settingsUI.py

```diff
@@ -0,0 +1,209 @@
+from abc import ABCMeta, abstractmethod
+from fbs_runtime.application_context.PyQt5 import ApplicationContext
+
+from PyQt5.QtGui import QPixmap
+from PyQt5.QtCore import Qt
+
+from ..widgets import (
+    Frame,
+    HorizontalLayout,
+    HorizontalSpacer,
+    Label,
+    SectionTitleFrame,
+    VerticalLayout,
+    VerticalSpacer,
+    ViewButton,
+)
+from ..config import VERSION
+
+
+class SettingsUI(Frame):
+
+    # metaclass for defining abstract base classes
+    __metaclass__ = ABCMeta
+
+    logo_section: Frame = None
+
+    content_section: Frame = None
+    faq_section: Frame = None
+    dashboard_section: Frame = None
+    resource_section: Frame = None
+    jobs_section: Frame = None
+
+    button_section: Frame = None
+
+    save: ViewButton = None
+    cancel: ViewButton = None
+
+    def __init__(self, cxt: ApplicationContext, *args, **kwargs):
+        super(SettingsUI, self).__init__(*args, name="settings", **kwargs)
+
+        self.cxt = cxt
+
+        self._init_ui()
+        self.setStyleSheet(self.cxt.settings_style)
+
+    @abstractmethod
+    def on_save_button_clicked(self):
+        pass
+
+    @abstractmethod
+    def on_cancel_button_clicked(self):
+        pass
+
+    def _init_ui(self):
+
+        window_layout = VerticalLayout(self)
+
+        self.logo_section = Frame(self, name="logo_section")
+        window_layout.addWidget(self.logo_section)
+        self._init_logo_section()
+
+        self.content_section = Frame(self, name="content_section")
+        window_layout.addWidget(self.content_section)
+        self._init_content_section()
+
+        self.button_section = Frame(self, name="button_section")
+        window_layout.addWidget(self.button_section)
+        self._init_button_section()
+
+        self.save.clicked.connect(self.on_save_button_clicked)
+        self.cancel.clicked.connect(self.on_cancel_button_clicked)
+
+    def _init_logo_section(self):
+
+        section_layout = HorizontalLayout(self.logo_section)
+
+        title = Label(
+            self.logo_section,
+            text="Setting",
+            name="setting_title",
+            align=Qt.AlignVCenter,
+        )
+        section_layout.addWidget(title)
+
+        spacer = HorizontalSpacer()
+        section_layout.addItem(spacer)
+
+        logo_frame = Frame(self.logo_section)
+        logo_layout = VerticalLayout(logo_frame, space=8)
+        section_layout.addWidget(logo_frame)
+
+        # logo
+        logo = Label(logo_frame)
+        logo.setPixmap(QPixmap.fromImage(self.cxt.logo))
+        logo.setAlignment(Qt.AlignHCenter)
+        logo_layout.addWidget(logo)
+
+        # version
+        version = Label(
+            logo_frame, text=f"PLUTO version: v {VERSION}", name="setting_version"
```

Comment:
Can this `PLUTO version: {VERSION}` or `PLUTO version: v{VERSION}`?

Reply:
i am thinking about maybe we should have somewhere gloablly to record this version #. Right now, there's one at `base.json` and `config.py` as well.
fixed.